configs/friend-path/friend-path.dockerfile

```diff
@@ -6,6 +6,8 @@ EXPOSE 8080
 # Setting the work direcory for our app.
 WORKDIR /app
 
+RUN apk update && apk add curl
+
```

Comment:
the healthcheck uses cURL.
cURL does not exist in the alpine version of node, had to add it manually